hgtv/views/playlist.py

```diff
@@ -181,20 +182,33 @@ def playlist_edit(channel, playlist):
     (Playlist, {'name': 'playlist', 'channel': 'channel'}, 'playlist'),
     permission='delete'
     )
+@render_with({'text/html': 'index.html.jinja2'}, json=True)
 def playlist_delete(channel, playlist):
-    return render_delete_sqla(playlist, db, title=u"Confirm delete",
-        message=u"Delete playlist '%s'? This cannot be undone." % playlist.title,
-        success=u"You have deleted playlist '%s'." % playlist.title,
-        next=channel.url_for())
+    if request.method == 'GET':
+        return {'playlist': dict(playlist.current_access())}
```

Comment:
Instead of doing this, just change the view to `@app.route('/<channel>/<playlist>', methods=['DELETE'])`. You don't even need to support `GET` here.

Reply:
This view has a GET template that shows the Delete prompt.